docs/api/rest_api/rest_api_usage/testing_rest_api.md

```diff
@@ -54,17 +51,34 @@ The following example of an AJAX call retrieves `ContentInfo` (that is, metadata
 
 To test it, copy-paste this code into your browser console alongside a page from your website (to share the domain):
 
-```javascript
-var resource = '/api/ibexa/v2/content/objects/52',
-    request = new XMLHttpRequest();
-
-request.open('GET', resource, true);
-request.setRequestHeader('Accept', 'application/vnd.ibexa.api.ContentInfo+json');
-request.onload = function () {
-    console.log(request.getAllResponseHeaders(), JSON.parse(request.responseText));
-};
-request.send();
-```
+=== "Fetch API"
+
+    ```javascript
+    let resource = '/api/ibexa/v2/content/objects/52';
+    
+    fetch(resource, {
+        headers: {'Accept': 'application/vnd.ibexa.api.ContentInfo+json'},
+    }).then((response) => {
+        console.log(...response.headers);
```

Comment:
I'm not sure if there is a need to console log headers?

Reply:
@dew326
It corresponds to the console log of `request.getAllResponseHeaders()` from previous example. I don't know why it was there. It might be an opportunity to show REST response headers `accept-patch` and `content-type`.